x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/AutoConfigureNode.java

```diff
@@ -942,14 +942,18 @@ private void notifyOfFailure(boolean inEnrollmentMode, Terminal terminal, Termin
         }
     }
 
-    @SuppressForbidden(reason = "Commons IO lib uses the File API")
+    @SuppressForbidden(reason = "We need to interact with the file system in order to perform auto configuration")
     private void deleteDirectory(Path directory) throws IOException {
-        FileUtils.deleteDirectory(directory.toFile());
+        Files.walk(directory)
+            .sorted(Comparator.reverseOrder()) // Ensures directory contents are deleted first
+            .map(Path::toFile)
+            .map(File::delete);
     }
 
-    @SuppressForbidden(reason = "Commons IO lib uses the File API")
+    @SuppressForbidden(reason = "We need to interact with the file system in order to perform auto configuration")
     private void moveDirectory(Path srcDir, Path dstDir) throws IOException {
-        FileUtils.moveDirectory(srcDir.toFile(), dstDir.toFile());
+        Files.copy(srcDir, dstDir, StandardCopyOption.COPY_ATTRIBUTES);
+        deleteDirectory(srcDir);
```

Comment:
I looked at the code in `FileUtils#moveDirectory` and it first tries a `#rename`.
What do you think if we copy-paste the `FileUtils#moveDirectory` in the `IOUtils` class and use that here?